app/models/audio_file.rb

```diff
@@ -4,14 +4,16 @@ class AudioFile < BaseModel
 
   include PublicAsset
 
-  UPLOADED_TO_S3           = "uploaded"               # audio on s3 in upload, not in audio_file yet
-  VALIDATION_IN_PROCESS    = "validating"             # audio copied from upload, now checking file
-  VALID_AUDIO              = "valid"                  # validation passed, need to create mp3s
-  INVALID_AUDIO            = "invalid"                # validation failed (dead end)
-  REMOTE_UPLOAD_COMPLETED  = "complete"               # dropbox or basic upload worked, need to create mp3s
-  TRANSFORM_IN_PROGRESS    = "creating mp3s"          # was valid or complete, now making mp3s
-  TRANSFORM_FAILED         = "creating mp3s failed"   # mp3 creation failed (dead end)
-  TRANSFORMED              = "mp3s created"           # mp3 creation worked, done! (dead end)
+  UPLOADED         = "uploaded"               # audio on s3 in upload, not in audio_file yet
```

Comment:
Line is too long. [92/80]<br>Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.